examples/mujoco/ppo/train_ppo.py

```diff
@@ -0,0 +1,199 @@
+"""A training script of PPO on OpenAI Gym Mujoco environments.
+
+This script follows the settings of https://arxiv.org/abs/1709.06560 as much
+as possible.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+import argparse
+import functools
+
+import chainer
+from chainer import functions as F
+from chainer import links as L
+import gym
+import gym.spaces
+import gym.wrappers
+import numpy as np
+
+import chainerrl
+from chainerrl.agents import PPO
+from chainerrl import experiments
+from chainerrl import misc
+
+
+def main():
+    import logging
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--gpu', type=int, default=0,
```

Comment:
Wouldn't the default be -1? or is 0 going to do the same?

Reply:
`--gpu 0` uses GPU 0, while `--gpu -1` uses CPU only. Setting the default to CPU also makes sense, but currently all the example set the default to 0, GPU 0.

Reply:
I see, no problem